extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -2,13 +2,18 @@
  * External dependencies
  */
 import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
 import { memo } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { STATE_ERROR } from '../constants';
+import { getColorClassName } from '../util';
```

Comment:
```suggestion
import { getColorClassName } from '../utils';
```

Could we rename this to plural. It's likely we may end up with more than one. It's odd. I know it was my fault - sorry!

Reply:
<3